apps/native/app/package.json

```diff
@@ -63,6 +65,7 @@
     "expo-notifications": "0.28.9",
     "intl": "1.2.5",
     "lodash": "4.17.21",
+    "metro-config": "0.81.0",
```

Comment:
~Let's add a low-priority follow-up task to try merging these dependencies into the root package.json. At least all shared dependencies.